packages/core/modules-sdk/package.json

```diff
@@ -36,6 +35,7 @@
     "resolve-cwd": "^3.0.0"
   },
   "peerDependencies": {
+    "@mikro-orm/core": "*",
```

Comment:
Q: shouldn't we use fixed versions of mikro orm packages to not end up having multiple versions and leading to unsuccessful instance of checks under the hood?

Reply:
I added it as an optional peer dependency everywhere, so none package manager will install it. And only `@medusajs/utils` will have the version fixed.
Same for `awilix`